src/code_manager/model/CssGenerator.js

```diff
@@ -121,8 +125,10 @@ export default Backbone.Model.extend({
       }
     });
 
-    if ((selectorStrNoAdd && found) || selectorsAdd || singleAtRule) {
-      const block = rule.getDeclaration();
+    if ((selectorStrNoAdd && found) || selectorsAdd || singleAtRule || rule.attributes.style) {
+      const entries = Object.entries(rule.attributes.style);
+      const styleStr = entries.length > 0 ? entries.map(([k, v]) => `${k}:${v}`).join(';') + ';' : '';
+      const block = rule.getDeclaration().length > 0 ? rule.getDeclaration() : styleStr;
```

Comment:
You're calling rule.getDeclaration() twice and honestly, I don't understand the point of this change